src/interface.c

```diff
@@ -84,12 +84,15 @@ static const char *config_keys[] =
 {
 	"Interface",
 	"IgnoreSelected",
+	"ActiveInterfaceOnly",
 	NULL
 };
-static int config_keys_num = 2;
+static int config_keys_num = 3;
```

Comment:
The *interface plugin* is very old and this line is one of the lines where it shows. Please remove the `NULL` from the `config_keys` array and use:
```c
static int config_keys_num = STATIC_ARRAY_SIZE (config_keys);
```